### PR TITLE
Show how to add a page title to routes (#1381)

### PR DESCRIPTION
**Pull request by jgwhite**
Addresses #1381

### CODE REVIEW

**Comment by jgwhite** [comment]
I’ve added this content as discussed in the a11y strike team meeting, but in working on it with my colleague @zwhitfield3 we’re somewhat concerned that although this *is* the simplest place to start it might raise more questions that it answers. For example, I can imagine someone using this code, but soon needing to incorporate data from the model in the page title, and running into caveats with the `activate` approach (or, indeed, any of the model hooks). The more I look at it, the more I think it would be better to simply promote the existing addons until we have the unified ember-title approach available.

That said, I think the testing snippet is still valuable no matter what, so perhaps we can keep that?

Keen to hear your thoughts.

**Comment by amyrlam** [comment]
@jgwhite 👋 I was at the a11y meeting this week and am on the Learning Team can help get this PR merged. (Have been fixing a CI issue on this repo that should be fixed, if you can do a rebase, from the changes in https://github.com/ember-learn/guides-source/pull/1395.) 

As for the content, I think just adding the testing portion is fine. Or we could keep it but massage the verbiage e.g. instead of "the simplest" to say something like "a simple way, with some gotchas". Will post this PR in #st-a11y.

Also, feel free to tag @ ember-learn/learning-core-team as a reviewer in the future if you want / crosspost in #dev-ember-learning, I can also assist as should be able to make a11y meetings

**Comment by MelSumner** [comment]
@jenweber in the other sections of the guides where I talk about A11y, I talk about how to do it in Ember right now (even if it's complex or not ideal) and then at the end list other addons that can help. It might be good to be consistent.

**Comment by jgwhite** [comment]
Thanks everyone for your input. I’ve taken it all into account and taken it in a slightly new direction.

@amyrlam mind giving this another look?

File: guides/release/accessibility/page-template-considerations.md [hunk 1]
```diff
@@ -15,14 +15,76 @@ Consider this format:
...
+
+  compute([title]) {
+    document.title = title;
+  }
+
+  willDestroy() {
+    document.title = this.original;
+  }
+}
+
```
Comment (amyrlam) [inline guides/release/accessibility/page-template-considerations.md:34]: Hard to apply in GitHub suggestion, but copy edit suggestion - delete blank line before ```

File: guides/release/accessibility/page-template-considerations.md [hunk 2]
```diff
@@ -15,14 +15,76 @@ Consider this format:
...
+This title will take effect when we enter the “post” route, and the line of code in our `willDestroy` hook will take care of restoring the former title when we return to the “posts” route.
+
+This technique is a reasonable first step, but has limitations:
+
+- It does *not* work when the page is rendered server-side with FastBoot.
+- It doesn’t provide any conventions for constructing nested page titles.
+- It doesn’t automatically apply the site title (though you can imagine how to add that).
+
+When your needs become more complex, the following addons faciliate page titles in a more dynamic and maintainable way (including FastBoot support):
```
Comment (amyrlam) [inline guides/release/accessibility/page-template-considerations.md:64]:
```suggestion
When your needs become more complex, the following addons facilitate page titles in a more dynamic and maintainable way (including FastBoot support):
```

File: guides/release/accessibility/page-template-considerations.md [hunk 3]
```diff
@@ -15,14 +15,76 @@ Consider this format:
...
+
+<h1>{{@model.title}}</h1>
+```
+
+This title will take effect when we enter the “post” route, and the line of code in our `willDestroy` hook will take care of restoring the former title when we return to the “posts” route.
+
+This technique is a reasonable first step, but has limitations:
+
+- It does *not* work when the page is rendered server-side with FastBoot.
+- It doesn’t provide any conventions for constructing nested page titles.
```
Comment (amyrlam) [inline guides/release/accessibility/page-template-considerations.md:61]:
```suggestion
- It doesn't provide any conventions for constructing nested page titles.
```

Not familiar with this linter yet, but seems to be giving a warning https://travis-ci.org/github/ember-learn/guides-source/builds/676236275#L545

I will set up these linters locally after this!

File: guides/release/accessibility/page-template-considerations.md [hunk 4]
```diff
@@ -15,14 +15,76 @@ Consider this format:
...
+- It doesn’t automatically apply the site title (though you can imagine how to add that).
```
Comment (amyrlam) [inline guides/release/accessibility/page-template-considerations.md:62]:
```suggestion
- It doesn't automatically apply the site title (though you can imagine how to add that).
```

File: guides/release/accessibility/page-template-considerations.md (hunk 2 above)
Comment (jgwhite) [reply guides/release/accessibility/page-template-considerations.md:64]: LOL, good spot!

**Comment by jgwhite** [comment]
Thanks for the review @amyrlam. I’ll attend to your feedback as soon as possible.

**Comment by jgwhite** [comment]
@amyrlam I’ve applied all those suggestions and added an extra caveat (might be overthinking this 😅)

**Comment by jgwhite** [comment]
> warning  `realtime` is misspelt; did you mean `mealtime`?

These listing rules are strict!

**Pull request merged**